src/daemon/http/server.rs

```diff
@@ -597,7 +595,7 @@ pub async fn rfc8181(req: Request) -> RoutingResult {
             Err(e) => return render_error(e),
         };
 
-        let read = state.read().await;
+        let read = state;
```

Comment:
Same here; just ditch `read` and keep using `state` below?

Reply:
indeed.. now removed. (to quick to search and remove unlocking the now removed RwLock)